tests/storage/guestfs.go

```diff
@@ -84,26 +85,11 @@ var _ = SIGDescribe("[rfe_id:6364][[Serial]Guestfs", func() {
 			defer GinkgoRecover()
 			Expect(guestfsCmd.Execute()).ToNot(HaveOccurred())
 		}()
-		// Waiting until the libguestfs pod is running
+		// Waiting until the libguestfs pod is ready
 		Eventually(func() bool {
 			pod, _ := virtClient.CoreV1().Pods(util.NamespaceTestDefault).Get(context.Background(), podName, metav1.GetOptions{})
-			ready := false
-			for _, status := range pod.Status.ContainerStatuses {
-				if status.State.Running != nil {
-					return true
-				}
-			}
-			return ready
-
+			return tests.PodReady(pod) == corev1.ConditionTrue
 		}, 90*time.Second, 2*time.Second).Should(BeTrue())
```

Comment:
nit: maybe better to read

```suggestion
		Eventually(func() corev1.ConditionStatus {
			pod, _ := virtClient.CoreV1().Pods(util.NamespaceTestDefault).Get(context.Background(), podName, metav1.GetOptions{})
			return tests.PodReady(pod)
		}, 90*time.Second, 2*time.Second).Should(BeEquivalentTo(corev1.ConditionTrue))
```

Reply:
Thanks, done